node/src/app.ts

```diff
@@ -28,8 +27,9 @@ export async function createApp(): Promise<Express> {
     app.use(morgan('dev'));
   }
   app.use(cors());
+  app.use(express.json({ limit: '50mb' }));
```

Comment:
the use cases we actually have are more like 200K, right? Maybe we should make these `1mb` for now?